.github/ISSUE_TEMPLATE.md

```diff
@@ -38,7 +38,7 @@
 // additional code, remove if not needed.
 ```
 
-### Please paste the results of `webpack-cli info` and `webpack-cli version` here, and mention other relevant information
+### Please paste the results of `npx --no-install webpack info` here, and mention other relevant information
```

Comment:
hm I try locally, and it still install `webpack-cli`, in docs `The --no-install option is deprecated, and will be converted to --no.`, but nothing to work :confused:

Reply:
https://docs.npmjs.com/cli/v7/commands/npx

Reply:
can you try `npx --no-install webpack-cli info`?